src/components/ChatLog.jsx

```diff
@@ -0,0 +1,39 @@
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry.jsx';
+
+const ChatLog = (props) => {
+  const messageComponents = props.entries.map(message => {
+    return (
+      <ChatEntry className="chat-log"
+        key={message.id}
+        id={message.id}
+        sender={message.sender}
+        body={message.body}
+        timeStamp={message.timeStamp}
+        liked={message.liked}
+        onLikeToggle={props.onLikeToggle}
+      ></ChatEntry>
```

Comment:
Since the names of the keys on `entry` are the same as the names your are setting on `ChatEntry` attributes/you are using all of the values in `entry` you could do something something like this to save you a few keystrokes:
```javascript
const  chatComponents  =  entries.map((entry)  =>  {
  return(
    <ChatEntry
	{...entry}
	onLikeToggle={onLikeBtnToggle}
	key={entry.id}
	/>
  );
});
```
Though, what you have is more verbose! The suggestion I have above can be used when we want to use pass every `key/value` pair to the child component. If not, then we wouldn't follow the suggested pattern since it would be passing impertinent information to a component, putting it at risk to accidentally change a piece and be more bug prone.